@xen-orchestra/web-core/lib/locales/fr.json

```
@@ -10,7 +10,7 @@
   "admin-login": "Nom d'utilisateur administrateur",
   "admin-password": "Mot de passe administrateur",
   "admin-password-confirm": "Confirmer le mot de passe administrateur",
-  "affinity-host": "Affinity host",
+  "affinity-host": "Affinité avec l'host",
```

What do you think of:
```suggestion
  "affinity-host": "Hôte de préférence",
```